docusaurus.config.js

```diff
@@ -43,6 +43,11 @@ const config = {
           editUrl:
             'https://github.com/jellyfish-dev/jellyfish-docs',
           routeBasePath: '/',
+          versions: {
+            current: {
+              label: '0.3.0-dev'
```

Comment:
Let's change this to something more generic so that we don't have to update it every time

Reply:
Oops, I closed the PR

Reply:
So i guess we can stick to `next`